lib/Remotery.c

```diff
@@ -189,6 +189,10 @@ static rmtBool g_SettingsInitialized = RMT_FALSE;
     #include <atomic>
 #endif
 
+#ifndef RMT_PLATFORM_WINDOWS
+    #define ZeroMemory(dest, len) memset(dest, 0, len)
```

Comment:
Can you just use `memset` in Vulkan code instead of overloadnig this symol?

Reply:
Definitely - I was just being lazy...